### ISSUE TITLE
rec-live did not works

### ISSUE THREAD

**Issue by e-jigsaw** [yyoshiki41/radigo#26]
```
docker run -it yyoshiki41/radigo radigo rec-live -id=TBS -t=1000
Now downloading..
+------------+---------------+
| STATION ID | DURATION(SEC) |
+------------+---------------+
| TBS        |          1000 |
+------------+---------------+
| ERROR: Failed to construct rtmpdump command: exec: "rtmpdump": executable file not found in $PATH
```

so I tried to install `rtmpdump` manually.

```
docker run -it yyoshiki41/radigo /bin/bash
apt-get install rtmpdump
radigo rec-live -id=TBS -t=1000
```

then

```
Now downloading..
+------------+---------------+
| STATION ID | DURATION(SEC) |
+------------+---------------+
| TBS        |          1000 |
+------------+---------------+
| ERROR: Failed to execute ffmpeg command: exit status 1
```

How to fix this problem?

**Comment by yyoshiki41** [comment]
@e-jigsaw 
Thanks for your report!

Maybe, this occurs only in the docker environment.
It takes time to fix.
So if you are in a hurry, try below installation in your local machine.
https://github.com/yyoshiki41/radigo#building-from-source

I'll prepare the docker develop enviroment and debug it.

**Comment by e-jigsaw** [comment]
Thank you for your reply.
I am looking forward to fix this issue!

**Comment by yyoshiki41** [comment]
Hi, @e-jigsaw 

Fixed this issue in `v0.6.0`
This issue caused by ffmpeg that does not support `libmp3lame`.

Please `docker pull` on latest tag and try `rec-live`!

Thanks 👍
